.travis.yml

```diff
@@ -12,11 +12,11 @@ env:
   global:
     - COMPOSE_FILE=./containers/docker-compose-testing.yml
   matrix:
-    - TASK="rake test:all"
-    - TASK="rails test test/unit"
-    - TASK="rails test test/functional"
-    - TASK="rails test test/integration"
-    - TASK="rails test:system"
+    - TASK="TEST_TYPE=mocha rake test:all"
```

Comment:
What if we removed the TEST_TYPE but looked for `TASK == "rake test:system"`?

Reply:
No, because the current output shows that `TEST` is not passed in. But maybe we can change that in the container config...